### PR TITLE
Testing rspec

### PR DESCRIPTION
**Pull request by gscarv13**
Ruby - Testing Your Code - Tic-Tac-Toe

- Initiated rspec testing on root rspec --init.
- Created testing_rspec branch for testing purposes.
- Created game_logic_spec.rb file for testing purpose.
- Created test scenarios to validate our code.

### CODE REVIEW

File: spec/game_logic_spec.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+  describe '#check_input' do
+    it 'returns false if input == 0' do
+      expect(board.check_input(0)).to be false
+    end
+
+    it 'returns false unless input.positive? && input < 10' do
+      expect(board.check_input(-2)).to be false
+      expect(board.check_input(12)).to be false
+      expect(board.check_input(5)).to be true
```
Comment (MarilenaRoque) [inline spec/game_logic_spec.rb]: - Separate each expect in one it block

File: spec/game_logic_spec.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  win2.change_box('1', player2)
+  win2.change_box('4', player2)
+  win2.change_box('7', player2)
+
+  draw = Board.new
+  draw.change_box('1', player1)
+  draw.change_box('2', player1)
+  draw.change_box('3', player2)
+  draw.change_box('4', player2)
+  draw.change_box('5', player2)
```
Comment (MarilenaRoque) [inline spec/game_logic_spec.rb:28]: - Build tests to check the behavior of that function

**Review by MarilenaRoque — CHANGES_REQUESTED** [review]
Hello Micronauts! :rocket:

Good job on your work on that project, so far! :clap:

However, there are a few necessary changes to complete all the requirements.

## Changes Required ♻️ 

- [x] Use one it block for each expected behavior. Do not test multiple aspects in one it block.

Kindly, wrap each <expect> scenario in one <it> block.

- [ ] Add specs for all public methods you created in Tic Tac Toe project.

Please, add tests for the #change_box method to cover all the public methods.

Kindly check my comments below for more details and the [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/) to check the requirements listed.

Additional kudos on the separation of concerns in your methods. 😄 

Happy Coding!!! 💻 

**_Marilena Roque
Full-Stack Developer
Code Reviewer at Microverse Inc.
[Portfolio](https://mariroqueproject.com/) | [Linkedin](https://www.linkedin.com/in/roquemarilena/) 
marilenas@id.uf.br_**

**Comment by pipe2442** [comment]
Thanks, @MarilenaRoque all the changes were implemented. 😄

**Review by Georjane — CHANGES_REQUESTED** [review]
## Status: Changes Required 🛑 🔧 
### Hi there Team ✋🏽 
Very nice work you have done with the rspec testing. You have implemented all changes required by the previous Code Reviewer. It's a thumbs up for me. 👍🏽 

- [x] However, kindly add instructions for testing to your Readme file to get this project approved.

_Feel free to reach out to me on Slack if you have any questions, Witah Georjane_ 🤝 
Good luck on your project and Happy Coding 💻 

**Witah Georjane
Code Reviewer at Microverse Inc
[portfolio](https://witahgeorjane.tech/) | [linkedIn](https://www.linkedin.com/in/witah-georjane/)**

**Review by teekaytech — APPROVED** [review]
### Status: Approved :green_circle: 

**Congratulations team, this project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:**

Again, congratulations, you can merge to the default branch.
Good luck with your next project.

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**